src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
 [assembly: AssemblyVersion("6.0.0.0")]
-[assembly: AssemblyFileVersion("6.0.1.0")]
+[assembly: AssemblyFileVersion("6.0.2.0")]
```

Comment:
Will revert this to 6.0.1 after talking to @markcowl. We will use the same version for both network SDK changes in this release.